CHANGELOG.md

```diff
@@ -5,7 +5,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
-- No changes yet.
+### Added
+- gen: Added `go.label` used in enum item to override enum
+  item name in codegen.
```

Comment:
i think we should be more explicit about what exactly `go.label` affects here. as someone without much context on this change, it's unclear what "item name in codegen" actually means. does it mean the name of the Go identifier? Does it mean the name in `String`? What about JSON/YAML?

Reply:
good point, what do you think of:
```
- gen: Added `go.label="<TAGGED_NAME>"` annotation for enum items. Corresponding items
  of the generated Go structs will be using <TAGGED_NAME> for JSON marshal/unmarshal
  instead of origin item name, this change does not apply to YAML
```

Reply:
I think this change applies to all text outputs -- `String()`, JSON, YAML and anything else that uses `TextMarshaler`

Reply:
I'll revise it to
```
- gen: Added `go.label="<TAGGED_NAME>"` annotation for enum items. 
  Corresponding items of the generated Go structs will be using 
  <TAGGED_NAME> for text marshal / unmarshal instead of its origin item
  name
```